### PR TITLE
Updates to contribution guidelines

### PR DESCRIPTION
**Pull request by omri374**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+## Issue reference
+This PR fixes issue #XX
+
+## Checklist
+- [ ] I have reviewed the [contribution guidelines](../../CONTRIBUTING.md)
+- [ ] I have signed the CLA
+- [ ] My code includes unit tests
+- [ ] All unit tests and lint checks pass locally
+- [ ] My PR contains documentation updates / additions if required
+- [ ] My code doesn't require a change in the project's Protobuf definition (github.com/microsoft/presidio-genproto) or if they do, I have submitted a PR for a protobuf change.
```
Comment (eladiw) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:13]: #YY protobuf pr link?

Comment (omri374) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:13]: You mean to ask them to provide the protobuf PR link in this pr?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,6 +21,14 @@ For more details follow the [Build and Release documentation](docs/build_release
 
 To get started, refer to the documentation for [setting up a development environment](docs/development.md).
 
+### General contribution guidelines
+
+- A Github issue suggesting the change should be opened prior to a PR.
+- All contributions should be documented, tested and linted. Please verify that all tests and lint checks pass successfully before proposing a change.
+- In order for a pull request to be accepted, the CI (containing unit tests, functional tests and linting) needs to succeed, in addition to approvals from two core contributors.
```
Comment (CodeRunRepeat) [inline CONTRIBUTING.md:28]: "core contributors" vs. "maintainers": is this intentional?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -21,6 +21,14 @@ For more details follow the [Build and Release documentation](docs/build_release
...
+- PRs should be small and solve/improve one issue at a time. If you have multiple suggestions for improvement, please open multiple PRs.
+
+
 ### How can I contribute?
 
 - [General contribution guidelines](#general-contribution-guidlines)
```
Comment (CodeRunRepeat) [inline CONTRIBUTING.md:34]: The general guidelines have been removed so the link should go too.

File: CONTRIBUTING.md (hunk 2 above)
Comment (omri374) [reply CONTRIBUTING.md:28]: No. Is there a difference?

**Pull request merged**